NWNXLib/Assert.cpp

```diff
@@ -0,0 +1,109 @@
+#include "Assert.hpp"
+
+#include <cstdlib>
+#include <cstring>
+
+#ifdef _WIN32
```

Comment:
`#if OS_WINDOWS` instead? (other instances in this file)

Reply:
This is a refactoring fail actually - I didn't define OS_WINDOWS in the project and changes it to _WIN32 but left  the // OS_WINDOWS comments here. I'll fix those comments.